lib/account/account-service.js

```diff
@@ -3,154 +3,80 @@
 /**
  * Handles the creating, deletion, and updating of account and
  * registration files.
- */
-
-/**
- * Information about an account holder. Used to generate/validate
- * a password and two factor authentication as well as ids and
- * contact information.
- *
- * @typedef {Object} accountService~accountFile
- * @property {string} email
- * @property {string} mfaKey
- * @property {string} password
- * @property {string} passwordSalt
- */
-
-/**
- * @typedef {Object} accountService~accountInfo
- * @property {string} accountId
- * @property {string} password
- */
-
-/**
- * Information sent to be saved in registration file.
  *
- * @typedef {Object} accountService~initiationFile
- * @property {string} email
- * @property {string} mfaKey
- * @property {string} passwordSalt
+ * @param {Object} config
+ * @param {Object} OtDate Helper object and Object class, like Date
+ * @param {Object} record
+ * @param {Object} storage
+ * @return {Function} factory
```

Comment:
I see an explanation of this module,AccountService, but this information is missing at the top of AccountManager, there's no explanation and parameter breakout for the manager.